ReleaseTooling/Sources/ZipBuilder/CarthageUtils.swift

```diff
@@ -42,31 +42,21 @@ extension CarthageUtils {
   static func packageCarthageRelease(templateDir: URL,
                                      artifacts: ZipBuilder.ReleaseArtifacts,
                                      options: CarthageBuildOptions) -> URL? {
-    guard let zipLocation = artifacts.carthageDir else { return nil }
+    guard let carthagePath = artifacts.carthageDir else { return nil }
 
     do {
       print("Creating Carthage release...")
-      let carthagePath =
-        zipLocation.deletingLastPathComponent().appendingPathComponent("carthage_build")
```

Comment:
Another copy of the carthage artifacts is not needed since they were already in the dedicated CarthageFirebase directory.